### DISCUSSION TITLE
There is an issue with [@types/mui-datatables]

### DISCUSSION THREAD

**Discussion by vcardins** [DefinitelyTyped/DefinitelyTyped#65499] [Issues with a @types package]
According to the code base, the `TableBodyRow` component accepts the [children](https://github.com/gregnb/mui-datatables/blob/e09c39cb3c5713c7c157845b3d3e4b7d77c77649/src/components/TableBodyRow.js#L78) prop,  
however the types does't contain it. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/mui-datatables/index.d.ts#L974

Expected Output:
Include the children prop to the TableBodyRow types definition.

[a comment by an automated account is not shown]

**Comment by favna** [comment]
Please make a pull request adding the missing property and I'll review it.
